src/jsMain/kotlin/process/phonemes/PhonemesMapping.kt

```diff
@@ -23,12 +26,29 @@ data class PhonemesMappingRequest(
 
     companion object {
 
-        fun findPreset(name: String) = Presets.find { it.first == name }?.second
+        fun findPreset(name: String) = Presets.find { it.name == name }?.phonemesMap
 
         fun getPreset(name: String) = requireNotNull(findPreset(name))
 
-        val Presets: List<Pair<String, PhonemesMappingRequest>> by lazy {
-            listOf()
+        val Presets: List<PhonemesMappingPreset> by lazy {
+            listOf(
+                PhonemesMappingPreset(
+                    sourceFormats = listOf(Format.Svp),
+                    targetFormats = Format.vocaloidFormats,
+                    name = "Japanese",
```

Comment:
I feel it's still better to display a full name of the mapping table used, for better references in case of inquiry, bug report etc.

Reply:
How about something like "Japanese (SynthV to Vocaloid)"? I want to keep the language first to make it easier to find. If needed I can also rename the text files to match this format.

Reply:
I think this name is good! The txt file names are not exposed to users, so the current names are fine